src/test/kotlin/uk/gov/justice/digital/hmpps/visitscheduler/integration/MigrateVisitTest.kt

```diff
@@ -109,18 +111,18 @@ class MigrateVisitTest : IntegrationTestBase() {
       assertThat(visit.visitContact!!.name).isNotEmpty
       assertThat(visit.visitContact!!.name).isEqualTo("John Smith")
       assertThat(visit.visitContact!!.telephone).isEqualTo("013448811538")
-      assertThat(visit.createTimestamp).isNotNull()
+      assertThat(visit.createTimestamp).isNotNull
       assertThat(visit.visitors.size).isEqualTo(1)
       assertThat(visit.visitors[0].nomisPersonId).isEqualTo(123)
-      assertThat(visit.visitNotes.size).isEqualTo(4)
-      assertThat(visit.visitNotes[0].type).isEqualTo(VISITOR_CONCERN)
-      assertThat(visit.visitNotes[1].type).isEqualTo(VISIT_OUTCOMES)
-      assertThat(visit.visitNotes[2].type).isEqualTo(VISIT_COMMENT)
-      assertThat(visit.visitNotes[3].type).isEqualTo(STATUS_CHANGED_REASON)
-      assertThat(visit.visitNotes[0].text).isEqualTo("A visit concern")
-      assertThat(visit.visitNotes[1].text).isEqualTo("A visit outcome")
-      assertThat(visit.visitNotes[2].text).isEqualTo("A visit comment")
-      assertThat(visit.visitNotes[3].text).isEqualTo("Status has changed")
+      assertThat(visit.visitNotes)
```

Comment:
your going to get merge issues here, this has changed

Reply:
Says it's OK to merge.